io-package.json

```diff
@@ -499,7 +499,7 @@
             "common": {
                 "name": "alarm",
                 "type": "boolean",
-                "write": true,
```

Comment:
why do you change the "write" flag here?
Is this object used at all currently?

Reply:
The alarm panel supports 5 statuses, 1 for disable and 4 for enable. An alarm=true cannot therefore be assigned directly, so from my point of view it does not make sense to write to this object. The alarm state can be set with the new object control.alarm.arm_state.